hadoop-tools/hadoop-azure-datalake/src/test/java/org/apache/hadoop/fs/adl/live/TestAdlSdkConfiguration.java

```diff
@@ -74,4 +75,67 @@ public void testDefaultTimeout() throws IOException {
 
     // The default value may vary by SDK, so that value is not tested here.
   }
+
+  @Test
+  public void testSSLChannelMode() throws IOException {
```

Comment:
this test goes through all the different configurations and then retries the same action through copy-pase (==maintenance cost). Either pull the common code into a method called repeatedly, or use parameterized junit tests